SimplenoteUITests/SimplenoteUITests.swift

```diff
@@ -0,0 +1,410 @@
+//
+//  SimplenoteUITests.swift
+//  SimplenoteUITests
+//
+//  Created by Xue Qin on 1/31/18.
+//  Copyright © 2018 Automattic. All rights reserved.
+//
+
+import XCTest
+
+class SimplenoteUITests: XCTestCase {
+    
+    let app = XCUIApplication()
+    let username = "ui.privacy.utsa4@gmail.com"
```

Comment:
Was this account created using the `history-analyst-dad` app id? It's a little weird if we have a username/password in the source code for security reasons.